SDK Showcase/SDK Showcase/AppState/AppState.swift

```diff
@@ -20,13 +20,25 @@ class AppState: ObservableObject {
     func unsetSystemError() {
         system.unsetError()
     }
+    
+    func resetRegisteredUsers() {
+        registeredUsers.removeAll()
+    }
+    
+    func addRegisteredUser(_ newUser: UserData) {
```

Comment:
something is wrong with the logic. When I registered another user I still see the only one user on the user list